src/components/Button/Button.js

```diff
@@ -3,8 +3,14 @@ import PropTypes from 'prop-types';
 
 import './button.css';
 
-const Button = ({ symbole, color, flex }) => (
-  <button type="button" style={{ background: `${color}`, flex: `${flex}` }}>
+const Button = ({
+  symbole, color, flex, handleClick,
```

Comment:
- [ ] Good job implementing logic, however I think it would be better to not pass styles property as a prop in your buttons component, you are only passing symbol and handleclick event handler to your buttons component inside your calculator component, using flex and color as prop also gives the error in the console please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/218791174-b7ed53bf-8d43-4b0c-89a2-bf8611f9d762.png)

I think it would be better if you rather pass a classname prop that will be the class for the buttons and use CSS to style them using that class like this :point_down: 
```
const Button = ({
  symbole, className, handleClick,
}) => (
  <button
    type="button"
    className={className}
    onClick={handleClick}
  >
    {symbole}
  </button>
);
```
you should also modify your proptypes check like this :point_down: 
```
Button.propTypes = {
  symbole: PropTypes.string.isRequired,
  className: PropTypes.string,
  handleClick: PropTypes.func.isRequired,
};

Button.defaultProps = {
  className: ''
}
```
this way we get rid of all the errors without breaking any functionality :+1: